NewHorizons/Assets/translations/portuguese_br.json

```diff
@@ -1,6 +1,69 @@
 {
-  "$schema": "https://raw.githubusercontent.com/xen-42/outer-wilds-new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "$schema": "https://raw.githubusercontent.com/outer-wilds-new-horizons/new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "DialogueDictionary": {
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_1": "Sua nave agora está equipada com uma unidade de translocação!",
```

Comment:
"Sua nave agora está equipada com uma unidade de dobra espacial!"

Reply:
The game's own translation file uses "translocar" instead of "dobra espacial" so I believe it to be more consistent with "translocar"

Reply:
Fair then 👍